rest-service/manager_rest/storage/resource_models_base.py

```diff
@@ -85,12 +88,18 @@ def tenant_name(cls):
     def created_by(cls):
         return association_proxy('creator', 'username')
 
+    @hybrid_property
+    def resource_availability(self):
```

Comment:
resource_availability is deprecated. 
for backwards compatibility - adding it to the response.